app/views/placements/support/schools/check.html.erb

```diff
@@ -0,0 +1,51 @@
+<%= content_for(:before_content) do %>
+  <%= govuk_back_link(href: new_placements_support_provider_path) %>
+<% end %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <%= form_with(model: @school, url: placements_support_schools_path, method: "post", data: { turbo: false }) do |f| %>
+      <%= f.hidden_field :urn, value: @school.urn %>
+
+      <label class="govuk-label govuk-label--l">
+        <span class="govuk-caption-l"><%= t(".add_organisation") %></span>
+        <%= t(".title") %>
+      </label>
+
+      <%= govuk_summary_list do |summary_list| %>
+        <% summary_list.with_row do |row| %>
+          <% row.with_key(text: t(".organisation_name")) %>
+          <% row.with_value(text: @school.name) %>
+        <% end %>
+        <% summary_list.with_row do |row| %>
+          <% row.with_key(text: t(".urn")) %>
+          <% row.with_value(text: @school.urn) %>
+        <% end %>
+      <% end %>
+
+      <h2 class="govuk-heading-m govuk-!-margin-top-7">
+        <%= t(".contact_details") %>
+      </h2>
+
+      <%= govuk_summary_list do |summary_list| %>
+        <% summary_list.with_row do |row| %>
+          <% row.with_key(text: t(".telephone")) %>
+          <% row.with_value(text: @school.telephone) %>
+        <% end %>
+        <% summary_list.with_row do |row| %>
+          <% row.with_key(text: t(".website")) %>
+          <% row.with_value(text: @school.website) %>
```

Comment:
Do you want the user to be able to click this? If so: 
```suggestion
         <% row.with_value do %>
            <%= govuk_link_to(@school.website, @school.website, target: "_blank", rel: "noopener noreferrer") %>
          <% end %>
```

Reply:
resolved in [URL improvements](https://github.com/DFE-Digital/itt-mentor-services/pull/73/commits/bcf290bc40d8af1d1efd2ba3013b830fa3160207). Had to add a helper to format external links in the `link_to` if the string doesn't have `http://` or `https://`